### PR TITLE
Add test for array builder

### PR DESCRIPTION
**Pull request by junli1026**
I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary

Add test for data array and builder 

## Changelog

- Testing

### CODE REVIEW

File: common/datavalues/src/arrays/builders/boolean_test.rs [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// Copyright 2020-2021 The Datafuse Authors.
```
Comment (BohuTANG) [inline common/datavalues/src/arrays/builders/boolean_test.rs:1]: We have the new license :)

Comment (junli1026) [reply common/datavalues/src/arrays/builders/boolean_test.rs:1]: will fix.

**Review by BohuTANG — APPROVED** [review]
LGTM
Thank you @junli1026

**Comment by BohuTANG** [comment]
CI failed, let's 'make lint'

**Comment by junli1026** [comment]
> CI failed, let's 'make lint'

Thanks for reminding. I updated the PR with formatted code. Should be fine now.

**Pull request merged**